<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Решебник задач</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(239, 239, 227);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 12pt &quot;Times New Roman&quot;;
background-color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rightLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);font: 75 italic 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Проверить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);font: 75 italic 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
